packages/venia-concept/src/components/PurchaseDetailsPage/OrderItem/__tests__/orderItem.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import OrderItem from '../orderItem';
+import ButtonGroup from '../ButtonGroup';
+
+configure({ adapter: new Adapter() });
+
+const itemMock = {
+    id: 1,
+    name: 'name',
+    size: '43',
+    color: 'color',
+    qty: 1,
+    titleImageSrc: 'picturePath',
+    price: 10,
+    sku: '24-MB01'
+};
+
+const classes = {
+    main: 'main',
+    imageAndPropsContainer: 'imageAndPropsContainer',
+    titleImage: 'titleImage',
+    propsColumnContainer: 'propsColumnContainer'
+};
+
+test('renders correctly', () => {
+    const onBuyAgainMock = jest.fn();
+    const onShareMock = jest.fn();
+
+    const wrapper = shallow(
+        <OrderItem
+            classes={classes}
+            item={itemMock}
+            onBuyAgain={onBuyAgainMock}
+            onShare={onShareMock}
+        />
+    ).dive();
+
+    expect(wrapper.find('.main')).toHaveLength(1);
+    expect(
+        wrapper
+            .find('.main')
+            .children()
+            .find('.imageAndPropsContainer')
```

Comment:
We can find .imageAndPropsContainer directly, and use values from classes object.

```
wrapper.find(`.${classes.imageAndPropsContainer}`)
```

Reply:
You did not fix this.